### ISSUE TITLE
Tournament Search return error object instead of empty list

### ISSUE THREAD

**Issue by toin0u** [RoyaleAPI/cr-api#276]
I was wondering if we could have an empty list instead of the error object because `404` and response body `[]` should be enough :)

## Request Details:
```
GET https://api.cr-api.com/tournaments/search?name=foobar
```

## Response Details:
HTTP response code: `404`
```
{
	"error": true,
	"status": 404,
	"message": "Not found | Additional information and support: http://discord.me/cr_api"
}
```

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there?
It has been always there.

 2. Expected behaviour: 
HTTP response code: `404`
```
[]
```

**Comment by fourjr** [comment]
/clan/search should be affected by this as well.

**Comment by selfish** [comment]
Fixed in `v10.6.9`
